controller/config/config.go

```diff
@@ -728,6 +744,46 @@ func LoadConfig(path string) (*Config, error) {
 
 	return controllerConfig, nil
 }
+func validateBindPoints(m map[interface{}]interface{}) []error {
```

Comment:
Is there a reason you didn't just ....use the parsed xweb during controller startup and do your validation there?

controller/controller.go line 232?

You have the parsed xweb instance and the controller identity.

Reply:
```

	c.xweb = xweb.NewDefaultInstance(c.xwebFactoryRegistry, c.config.Id)
	
	xcfg := c.xweb.GetConfig()
	for _, svcConfig := range xcfg.ServerConfigs {
		for _, bindPoint := range svcConfig.BindPoints {
			svcCertChains := c.config.Id.VerifyFor(bindPoint.Address)
		}
	}
``` 

?

Reply:
Mmm actually this is self-contained within xweb's config section. It can just go in xweb.

Reply:
just didn't know any better way.  i've moved it to xweb anyway and will push shortly